python/ray/tests/test_actor_failures.py

```diff
@@ -49,7 +49,7 @@ def create_object(self, size):
     # Submit enough methods on the actor so that they exceed the size of the
     # object store.
     objects = []
-    num_objects = 20
+    num_objects = 40
```

Comment:
20 doesn't work due to memory fragmentation (60MB chunks in 150MB => lots of fragmentation).